docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -156,11 +156,9 @@ Data Asset metrics provide you with insight into the data you can use for your d
 
 2. Click the **Overview** tab.
 
-3. Select one of the following options: 
-
-    - If you have not previously generated Data Asset metrics, click **Fetch Metrics**. 
+    When you select a new Data Asset, abbreviated Data Asset metrics are automatically generated.
 
-    - If you previously generated Data Asset metrics, click **Refresh** to refresh the metrics.
+3. Optional. Click **Profile Data** to refresh the Data Asset metrics or return all available metrics for a Data Asset.
```

Comment:
🚀

Reply:
Clarification: the button says "Profile Data" before additional metrics are calculated the first time. After that the button says "Refresh" as usual